src/health_misinfo_shared/fine_tuning.py

```diff
@@ -425,6 +427,27 @@ def construct_in_context_examples(data_filenames: list[str]) -> tuple[str, list[
     return examples, hold_out_set
 
 
+def infer_claims(video_id: str, transcript: list[dict]) -> Generator:
+    """For use in app"""
```

Comment:
To get this to work locally, I need to add the following here:
```python
vertexai.init(project=GCP_PROJECT_ID, location=GCP_TUNED_MODEL_LOCATION)
```

Is that something that’s missing here? Or is this a problem of my local setup?

Reply:
I presume I must have a default project set up on my system somewhere, which differs from `GCP_PROJECT_ID`.

For this to work in production, though, I suspect we do need this extra line.

Reply:
I'm going to tentatively assume project should be pulled from the metadata server and so is a you problem, while location might be something we want to add as an argument.

Reply:
Aha, okay – so I think if I run the `gcloud` CLI I can see where this project is currently specified:

```shell
$ gcloud config list

[core]
account = andy.lulham@fullfact.org
disable_usage_reporting = True
project = fullfact-development-2
```

I guess I can override this by adding `GOOGLE_APPLICATION_CREDENTIALS` to my local .env file.

There ought to be something in the README about this, because it’s not obvious currently.

Reply:
I don't know if this project's readme is the right place because this happens in any project where we have a Google cloud library using a GCP project ID, just usually that service is available on production and dev in all regions - but you are right it should be written down somewhere

Reply:
My immediate preference would be to add this line, because it’s a quick fix and it’s what we do elsewhere in this project already anyway, e.g.:
https://github.com/FullFact/health-misinfo-shared/blob/840e2a939f82b65e1519ed1644436098239b1c8f/src/health_misinfo_shared/vertex.py#L28

But I definitely agree with Dakota that hardcoding the project ID like this is not ideal. I think this ought to be set in the environment.

Reply:
Discussed with Dakota and we’re in agreement (that this is okay for MVP but not ideal) so I’ve pushed this change.

Reply:
For the MVP, I'm happy to hard code these values as they won't/can't change. But definitely one to plan more robustly when we build the actual prodct.